scripts/gdb-printers.py

```diff
@@ -77,10 +77,21 @@ def data(self):
         return self.val['m_pointer']
 
 
+class MallocBufferPrinter(BufferPrinter):
```

Comment:
I've never used these but it seems really cool. I don't imagine there's a good way to add a test, but would you mind at some point adding a short blurb to the sphinx? Like do I need to do anything to use it?

Reply:
Yes, I suppose a section under https://lvarray.readthedocs.io/en/latest/developmentAids.html would be appropriate

Reply:
Can you add it to the list of things on that page? Then once I get around to filling it out I might ask you to fill it in. Thanks.

Reply:
I ended up writing a few paragraphs about pretty printers, feel free to later use it however you see fit